test/indent.txt

```diff
@@ -19,12 +19,74 @@ to go
 ++]
 end
 
+IfMultiBracket1
+to foo
+  let abc 0
+  if true [ set abc [
+    1 2 3 ]
+  ] [
+++  set abc [4 5 6]
+  ]
+end
+
+# I'm not certain this should be normative
+# This behavior wasn't specified prior to 6.0.1 and could probably change
+IfMultiBracket2
```

Comment:
Related problem. Currently, this happens:

```
to foo
  let abc 0
  if true [ set abc
    [
      1 2 3
--    ] ]
...
```

which is wrong. Also, adding an open bracket on the end then does this:

```
to foo
  let abc 0
  if true [ set abc
    [
      1 2 3
++    ] ] [
```

This also seems wrong, as an open bracket shouldn't change the current line's indentation level. `++` should happen whether or not the open bracket is there.